test-forms/src/main/resources/forms/fieldlist-updates.xml

```diff
@@ -160,6 +160,17 @@
             <source15/>
             <target15/>
           </search_in_field_list_group>
+          <long_list_of_questions>
```

Comment:
This test relies on the not all questions being able to fit on screen right? I think if that's the case we should make this list absurdly long to prevent false positives on bigger devices. Maybe 100 questions? I think we should also add a comment that the list is designed to be longer than the screen so that's clear.

Reply:
> This test relies on the not all questions being able to fit on screen right?

Right.

I've just made the list longer (20 questions). I don't think we need more especially now as every question takes more space vertically compared to v2023.3.1.